.github/workflows/code-review.yaml

```diff
@@ -0,0 +1,51 @@
+# This workflow will perform the code review on each code change.
+# It uses "actions/cache" and turborepo in order to perform tasks only on changed files.
+
+name: "Code Review"
+
+on:
+  push:
+    branches: ["main"]
+    paths:
+      - apps/**
+      - packages/**
+  pull_request:
+    types: [opened, synchronize]
+    paths:
+      - apps/**
+      - packages/**
+  workflow_dispatch: {}
+
+jobs:
+  code_review:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
```

Comment:
```suggestion
        uses: actions/checkout@v3
```
better use hash

Reply:
You mean something like `actions/checkout@1f9a0c22da41e6ebfa534300ef656657ea2c6707`?

Reply:
correct